aws-params-reader-policy/main.tf

```diff
@@ -30,6 +29,9 @@ data "aws_iam_policy_document" "policy" {
     actions   = ["kms:Decrypt"]
     resources = [data.aws_kms_alias.parameter_store_key.target_key_arn]
   }
+  statement {
+    actions = ["ssm:DescribeParameters"]
+  }
 }
 
 resource "aws_iam_role_policy" "policy" {
```

Comment:
⚠️ **[tflint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Missing version constraint for provider "aws" in "required_providers"